Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -0,0 +1,342 @@
+//
+//  BRSystem.swift
+//  BRCrypto
+//
+//  Created by Ed Gamble on 3/27/19.
+//  Copyright © 2019 Breadwinner AG. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//  See the CONTRIBUTORS file at the project root for a list of contributors.
+//
+import Foundation
+import BRCryptoC
+///
+/// System (a singleton)
+///
+public final class System {
+
+    /// The listener.  Gets all events for {Network, WalletManger, Wallet, Transfer}
+    public private(set) weak var listener: SystemListener?
+
+    /// The account
+    public let account: Account
+
+    /// The path for persistent storage
+    public let path: String
+
+    /// The 'blockchain DB' to use for BRD Server Assisted queries
+    public let query: BlockChainDB
+
+    internal let queue = DispatchQueue (label: "Crypto System")
+
+    /// the networks, unsorted.
+    public internal(set) var networks: [Network] = []
+
+    ///
+    /// Add `network` to `networks`
+    ///
+    /// - Parameter network: the network to add
+    ///
+    internal func add (network: Network) {
+        networks.append (network)
+        announceEvent (SystemEvent.networkAdded(network: network))
+    }
+
+
+    /// The system's Wallet Managers, unsorted
+    public internal(set) var managers: [WalletManager] = [];
+
+    internal func managerBy (core: BRCryptoWalletManager) -> WalletManager? {
```

Comment:
Don't think these are used anymore.

Reply:
Axed the {BTC, ETH} related; left the BRCryptoWalletManager one for now (still feels to me that it might be useful; although, admittedly, the primary use **was** in the System implementations for callbacks where are now implemented in WalletManager...)